python/nav/web/maintenance/urls.py

```diff
@@ -32,6 +32,7 @@
     re_path(r'^active/$', views.active, name='maintenance-active'),
     re_path(r'^planned/$', views.planned, name='maintenance-planned'),
     re_path(r'^historic/$', views.historic, name='maintenance-historic'),
+    re_path(r'^search/$', views.component_search, name='maintenance-crazy-shit'),
```

Comment:
Why stick with `re_path`? `path` is easier to read.

Reply:
I could ask you the same.  I used the same pattern as the surrounding lines.  They were all last modified by you as part of some Django upgrade, I believe :)